### PR TITLE
onboarding: finished doc quiz

### PR DESCRIPTION
**Pull request by RosaGao**
## Description
Completed doc quiz

### CODE REVIEW

File: dev/readthedocs.md [hunk 1]
```diff
@@ -4,28 +4,43 @@ Please visit the [docs](https://semesterly-v2.readthedocs.io/en/latest/index.htm
 answer the following questions.
 
 1. What is the command I run to get the courses from Fall 2021?
+   python manage.py ingest jhu --years 2021 --terms fall
 
 2. How do I then load those courses into my database?
+   python manage.py digest jhu --years 2021 --terms fall
```
Comment (JJamesWWang) [inline dev/readthedocs.md:10]: You don't have to specify the terms/years, you can just do `python manage.py digest jhu`

File: dev/readthedocs.md [hunk 2]
```diff
@@ -4,28 +4,43 @@ Please visit the [docs](https://semesterly-v2.readthedocs.io/en/latest/index.htm
...
 
 3. How do I get a terminal running in my docker container?
+   docker-compose up
 
 4. Where do I store data I don’t want to commit?
+   add to .gitignore
 
 5. What is our stack?
+   Our stack includes Database, Backend and Frontend Frameworks, and CSS Framework.
```
Comment (JJamesWWang) [inline dev/readthedocs.md:19]: More specifically, what are these frameworks?

File: dev/readthedocs.md [hunk 3]
```diff
@@ -4,28 +4,43 @@ Please visit the [docs](https://semesterly-v2.readthedocs.io/en/latest/index.htm
...
    What about if I want to refactor myreducer?
+      refactor/refactor-myreducer
 
 8. What is the preferred format for commit messages?
+   Topic: Message
+   e.g.: 
+      Refactor Redux: refactor myreducer to myslice
 
 9. What linters do we run against our code?
+   ESLint
```
Comment (JJamesWWang) [inline dev/readthedocs.md:36]: There's two more, one more for frontend, and one for backend. What are they?

File: dev/readthedocs.md [hunk 4]
```diff
@@ -4,28 +4,43 @@ Please visit the [docs](https://semesterly-v2.readthedocs.io/en/latest/index.htm
 answer the following questions.
 
 1. What is the command I run to get the courses from Fall 2021?
+   python manage.py ingest jhu --years 2021 --terms fall
 
 2. How do I then load those courses into my database?
+   python manage.py digest jhu --years 2021 --terms fall
 
 3. How do I get a terminal running in my docker container?
+   docker-compose up
```
Comment (JJamesWWang) [inline dev/readthedocs.md:13]: How can you get a terminal in the container after you run `docker-compose up`?

File: dev/readthedocs.md [hunk 5]
```diff
@@ -4,28 +4,43 @@ Please visit the [docs](https://semesterly-v2.readthedocs.io/en/latest/index.htm
...
 
 4. Where do I store data I don’t want to commit?
+   add to .gitignore
```
Comment (JJamesWWang) [inline dev/readthedocs.md:16]: What about things like passwords or secrets? Where should they go?

**Review by JJamesWWang — COMMENTED** [review]
Sorry took me a while, I just forgot. I've added some follow up questions; anything that doesn't have a follow-up is good.

File: dev/readthedocs.md (hunk 1 above)
Comment (RosaGao) [reply dev/readthedocs.md:10]: Get it!

File: dev/readthedocs.md (hunk 4 above)
Comment (RosaGao) [reply dev/readthedocs.md:13]: We can create a new terminal and run commands in that separate window. Hope I am understanding the question right.

File: dev/readthedocs.md (hunk 5 above)
Comment (RosaGao) [reply dev/readthedocs.md:16]: Passwords or secrets should be added to semesterly/sensitive.py which is kept private and gitignored.

File: dev/readthedocs.md (hunk 4 above)
Comment (JJamesWWang) [reply dev/readthedocs.md:13]: Not sure either, so I'll explain:

Docker pretty much creates its own environment (the "container") for code to be run. So you have your computer, and your computer has a docker container, and its inside the container that the local server is run (not on your actual computer). By creating a new terminal, you would end up with a terminal at the $HOME directory of your computer, not inside the docker container. The question is then, how do you create a new terminal linked to the docker container, and not your computer? Feel free to ask if something doesn't make sense

File: dev/readthedocs.md (hunk 2 above)
Comment (RosaGao) [reply dev/readthedocs.md:19]: We use PostgreSQL with Django ORM for the database, Django with pycodestyle linter and Black formatter for backend, React/Redux with ESLint linter and Prettier formatter for frontend, and SCSS preprocesor with the BME methodology and Airbnb style guide for CSS.

File: dev/readthedocs.md (hunk 3 above)
Comment (RosaGao) [reply dev/readthedocs.md:36]: Frontend uses ESlint and Prettier, and Backend uses pycodestyle/Black.

Comment (RosaGao) [reply dev/readthedocs.md:36]: Correct me if Im wrong but are these like the IntelliJ CheckStyle to make code neat and consistent? Is there anything in particilar we should consider while writing code, or should we be running the linters in the end and make adjustments accordingly?

File: dev/readthedocs.md (hunk 4 above)
Comment (RosaGao) [reply dev/readthedocs.md:13]: Thanks for the clarification. We can use docker-compose run web /bin/bash to get a terminal in the container and use docker-compose run container-name commands. Is this correct?

Comment (RosaGao) [reply dev/readthedocs.md:13]: or is it docker exec -it container-name /bin/bash ?

Comment (JJamesWWang) [reply dev/readthedocs.md:13]: I know there *is* a way to do it with the command line (probably the 2nd one), but I don't know it. The way I intended for you to do it is through the VS Code extension. See if you can find where I mentioned it in the docs

File: dev/readthedocs.md (hunk 3 above)
Comment (JJamesWWang) [reply dev/readthedocs.md:36]: Yes, these are like CheckStyle and ensures the code is consistent and standardly formatted. When writing code, we try to be as clear as possible and write clean code. If you want to learn more, there's a well known book conveniently called Clean Code that I recommend (first 10 chapters or so)

File: dev/readthedocs.md (hunk 4 above)
Comment (RosaGao) [reply dev/readthedocs.md:13]: Sorry for the late reply. After docker-compose up, we right click on a container tab and attach a shell to the running container.

File: dev/readthedocs.md (hunk 3 above)
Comment (RosaGao) [reply dev/readthedocs.md:36]: Thanks. Will check this out when I have time!

**Review by JJamesWWang — APPROVED** [review]
Looks good, let me know when you want to try your first task

**Pull request closed without merging**